main.go

```diff
@@ -120,12 +123,38 @@ func sendEmail(event *types.Event) error {
 		return bodyErr
 	}
 
-	msg := []byte("To: " + toEmail + "\r\n" +
-		"Subject: " + subject + "\r\n" +
-		"\r\n" +
-		body + "\r\n")
+	if insecure {
+		msg := "To: " + toEmail + "\r\n" +
```

Comment:
Consider defining `msg` outside of this if block. It is duplicately defined on line 150, which can also be removed if it's defined earlier.

Reply:
Updated to move that out.